Classes/Http/FileServeStrategy/ReadfileStrategy.php

```diff
@@ -6,7 +6,8 @@
  *                                                                             */
 
 use Neos\Flow\Annotations as Flow;
-use Neos\Flow\Http\Response as HttpResponse;
+use Psr\Http\Message\ResponseInterface as HttpResponseInterface;
+use function GuzzleHttp\Psr7\stream_for;
```

Comment:
Doesn't this introduce a new dependency to guzzle? (I'm fine with adding that, but it should be added in the composer manifest, too)

Reply:
Thank you for your feedback. I added the guzzlehttp/psr7 dependency to the composer.json to be able to use the stream functions.